src/dataspec/base.py

```diff
@@ -382,37 +382,22 @@ def validate(self, v) -> Iterator[ErrorDetails]:
 
     @classmethod
     def from_validators(
-        cls,
-        tag: Tag,
-        *preds: Union[ValidatorFn, "ValidatorSpec"],
-        conformer: Optional[Conformer] = None,
+        cls, tag: Tag, *preds: ValidatorFn, conformer: Optional[Conformer] = None,
```

Comment:
How come this doesn't need to accept `ValidatorSpec`s anymore?

Reply:
The problem was basically that when we wrapped all of these other `ValidatorSpecs` in another `ValidatorSpec` we ended up getting an extra tag for every inner Spec failure. That wasn't helpful to anyone, since no one knew what the inner tags were _and_ `ValidatorSpec.from_validators` can only be called internally (it's not part of the public API).